tests/uiTests/loginTests.js

```diff
@@ -0,0 +1,125 @@
+const webdriver = require('selenium-webdriver');
+const chai = require('chai');
+require('dotenv').config();
+
+const { expect } = chai;
+
+let driver;
+
+const userDetails = {
+  username: process.env.DB_User_Username,
+  password: process.env.DB_User_Pw,
+};
+
+const adminDetails = {
+  username: process.env.DB_Admin_Username,
+  password: process.env.DB_Admin_Pw,
+};
+
+describe('Login', function () {
+  this.timeout(0);
+  beforeEach(async () => {
+    driver = await new webdriver.Builder().forBrowser('chrome').build();
+  });
+
+  afterEach(async () => {
+    await driver.quit();
+  });
+
+  it('should give appropiate menssage if incorrect userame provided', async () => {
```

Comment:
just a typo in the message '' should give appropriate message if incorrect username provided.

Reply:
done